symengine/symengine_config_cling.h.in

```diff
@@ -0,0 +1,7 @@
+#ifndef SYMENGINE_CONFIG_CLING_HPP
+#define SYMENGINE_CONFIG_CLING_HPP
+
+#pragma cling add_library_path(@SYMENGINE_CLING_LIBRARY_DIR@)
```

Comment:
Since it's just 2 lines, shall we just move them to `symengine/symengine_config.h.in`?

Reply:
The motivation for separating it in other libraries where we included this was that this relies on `cmake`'s templating mechanism (configure .in file).

In most cases, you want your package to be usable without cmake, so the the non-cling case, `symengine_config.h` should not be generated. I want to open another PR where the cmake version is fetched from `symengine_config.h` instead of `symengine_config.h` being generated from the version set in cmake.